osutil/cp_linux_test.go

```diff
@@ -36,11 +36,25 @@ func (s *cpSuite) TestCpMulti(c *C) {
 	c.Check(s.f2, testutil.FileEquals, s.data)
 }
 
+func (s *cpSuite) getReadOnlyFd(c *C) *os.File {
+	c.Check(osutil.CopyFile(s.f1, s.f2, osutil.CopyFlagDefault), IsNil)
+	c.Check(os.Chmod(s.f2, 0444), IsNil)
+	f2, err := os.Open(s.f2)
```

Comment:
os.Open() already opens the file "read-only" so the subsequent chmod is not needed.